### ISSUE TITLE
Question about joint mapping between smpl and smpl-x

### ISSUE THREAD

**Issue by ghdgkskghdgksk** [facebookresearch/frankmocap#91]
Thank you for your awesome work and I have a question about your paper.

You mentioned "Since all these annotations are in SMPL format, we ignore the shape parameters of ground truth SMPL annotations, and only use the pose parameters and 2D keypoint annotations that are compatible to SMPL-X model. W" in 3.3 3D Body Estimation Module.

To check this, I rendered the default poses of SMPL and SMPLX as shown in the figure below.
![unnamed](https://user-images.githubusercontent.com/82434566/114505402-4bad1480-9c6b-11eb-81b6-f384df4eeb2f.png)

Most of the joints except the hands and a head(15) were in the same position. 

My question is whether you used the 15th Head joint as the SMPL-X annotation. If so, isn't there an error according to the head position?

**Comment by penincillin** [comment]
@ghdgkskghdgksk 
Yes, you are right, Head (15) are different across SMPL and SMPL-X. 
@jhugestar Han, have you used head (15) in training the body-module (SMPL-X based) ?

**Comment by penincillin** [comment]
@ghdgkskghdgksk 
It seems that Han is too busy to answer this question in person. I will try to sync with him later and hopefully can resolve your issue.
Besides the joint definition, you can consider to re-train the SMPL-X based body module by transforming the SMPL parameters into SMPL-X format, using [official tools ](https://github.com/vchoutas/smplx/tree/master/transfer_model)offered by MPI.

**Comment by narratorxu** [comment]
@ghdgkskghdgksk
would you please share more clear image about the joints in smplx?  I also want to transform the SMPL parameters into SMPL-X format manually. Many thanks!
